nsv13/code/modules/jobs/nsv13_job_mail.dm

```diff
@@ -0,0 +1,58 @@
+//ATC
+/datum/job/air_traffic_controller
+	mail_goodies = list(
+		/obj/item/pizzabox/infinite = 20,
+		/obj/item/toy/cards/deck = 15,
+		/obj/item/clothing/glasses/sunglasses/advanced/gar = 10,
+		/obj/item/toy/katana = 8,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+	)
+
+//Bridge Officer
+/datum/job/bridge
+	mail_goodies = list(
+		/obj/item/pizzabox/infinite = 20,
+		/obj/item/clothing/glasses/sunglasses/advanced/garb = 10,
+		/obj/item/reagent_containers/food/drinks/ftliver = 8,
+		/obj/item/megaphone = 7,
+		/obj/item/paicard = 5,
+	)
+
+//Master At Arms
+/datum/job/master_at_arms
+	mail_goodies = list(
+		/obj/item/ship_weapon/ammunition/naval_artillery/cannonball = 15,
+		/obj/machinery/nuclearbomb/beer = 10,
+		/obj/item/ship_weapon/ammunition/naval_artillery/armed = 10,
+		/obj/item/ship_weapon/parts/missile/warhead/plushtide = 10,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/torp_juice = 8,
+		/obj/item/banner/munitions = 5,
+		//ITS CODED, WE SHOULD USE IT
```

Comment:
Never EVER use this as justification for adding anything, you don't know why it is or isn't used in any context.